src/views/tables/PositionsTable.tsx

```
@@ -460,19 +463,33 @@ export const PositionsTable = forwardRef(
 
     // todo this uses the old subaccount id for now
     const isAccountViewOnly = useAppSelector(calculateIsAccountViewOnly);
-    const marketSummaries = orEmptyRecord(useAppSelector(BonsaiCore.markets.markets.data));
-
-    const openPositions =
+    
+    // Use position processor data if available, otherwise fall back to real data
+    const { isMockMode } = usePositionProcessor();
+    const mockPositions = useMockPositions(currentMarket, marketTypeFilter);
+    const mockMarketSummaries = useMockMarketSummaries();
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)